src/app/harbor/shop/progress.tsx

```diff
@@ -19,19 +19,25 @@ export default function Progress({ val, items }) {
     return null
   }
 
+  const progressPercentage =
+    currentTix <= max.priceGlobal ? (currentTix * 100) / max.priceGlobal : 100
+
   return (
     <>
       <div className="relative ">
         <div className="relative w-full h-2.5 bg-white rounded overflow-visible">
           <div
             className="w-0 h-full bg-blue-500 rounded transition-all duration-300"
             style={{
-              width:
-                (currentTix <= max.priceGlobal
-                  ? (currentTix * 100) / max.priceGlobal
-                  : '100') + '%',
+              width: progressPercentage + '%',
             }}
           ></div>
+          <div
+            className="absolute top-0 left-1/2 transform -translate-x-1/2 text-white font-bold"
+            style={{ zIndex: 1 }}
+          >
+            {Math.round(progressPercentage)}%
+          </div>
```

Comment:
![image](https://github.com/user-attachments/assets/b8013031-5e14-404e-bca6-3fab9fd46dd7)

The reason this wasn't there before is that it's not just the *last* thing you're working towards, there are other things as well

I wonder if there's a way to make it consistent for all bookmarked items...